code/src/index.css

```diff
@@ -1,13 +1,160 @@
+/* GENERAL STYLING */
+* {
+box-sizing: border-box; 
+}
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: #000;
+  color: #fff;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* STYLING OF MAIN PAGE */
+
+.list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.list a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.movie-box {
+  width: 100%;
+  text-align: center;
+  position: relative;
+}
+
+.info-box {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  text-align: start;
+  margin: 20px;
+  overflow-wrap: break-word;
+}
+
+.movie-poster {
+  width:100%
+}
+
+/* STYLING OF SECOND PAGE */
+
+.more-page {
+  position: relative;
+}
+
+.background {
+  background-size: cover;
+}
+
+.poster {
+  border: 5px solid #fff;
+}
+
+.more-info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 100px 10px;
+}
+
+.more-info-text {
+  margin: 20px;
+  max-width: 400px;
+}
+
+.highlight {
+  margin: 5px;
+  font-weight: 300;
+  font-size: smaller;
+  color: red;
+}
+
+button {
+  position: absolute;
+  top: 50px;
+  left: 10px;
+  border: none;
+  background: none;
+  font-size: 20px;
+  color: white;
+  font-weight: 700;
+}
+
+/* MEDIA QUERIES */
+
+@media (min-width: 667px) {
+  .list {
+    flex-wrap: wrap;
+    flex-direction: row;
+  }
+
+  .movie-box {
+    width: 50%;
+  }
+
+  button {
+    left: 50px;
+  }
+
+  .more-info {
+    flex-direction: row;
+    align-items: flex-end;
+    padding: 100px 50px;
+  }
+}
+
+@media (min-width: 1024px) {
+  .movie-box {
+    width: 25%;
+  }
+
+  /* OVERLAY AND HOVER EFFECT STYLING ONLY ON DESKTOP VIEW */
```

Comment:
Nice job with the ease-in hover! I appreciate that you apply hover effects only on desktop to avoid weird mobile things.